### ISSUE TITLE
Can't create review for private repo

### ISSUE THREAD

**Issue by mtlynch** [codeapprove/feedback#5]
I'm trying to start a CodeApprove review for my first private repo, but CodeApprove fails with this error message:

>There is already a review for these branches on GitHub, but it does not have the codeapprove label: [view #41 on GitHub](https://github.com/tiny-pilot/gatekeeper/pull/41)

![image](https://user-images.githubusercontent.com/7783288/177551877-61d52dff-e266-4214-897c-5216f3770c3e.png)

It's saying I don't have the `codeapprove` label, but the label appears for me:

![image](https://user-images.githubusercontent.com/7783288/177552143-d897dc18-1db2-46a5-b9f3-9be2c6faa464.png)

**Comment by hatboysam** [comment]
@mtlynch thanks for this feedback. That feels like maybe there was a webhook failure somewhere ... does adding and removing the label help?

I'll definitely look into why that label check did not work as well.

**Comment by mtlynch** [comment]
@hatboysam - Ah, that was it. Removing and re-adding the label worked.

Now that you mention it, when I first tried to add the label on Github, I had to create `codeapprove` as a new label. I did "create and add label" but then the label didn't apply. I refreshed and then added the label a second time, and then it showed up on Github. So some failure there might have thrown CodeApprove into a weird state.

Stepping back though, once I'm already on the "Create Pull Request" screen in CodeApprove, is the `codeapprove` label a requirement? If CodeApprove is already looking at the PR where I want the review, what stops it from creating the review without me manually adding a label on the Github side?
